src/components/Footer/Footer.js

```diff
@@ -10,9 +10,22 @@ import MemoIc_block_inactive from '../../assets/block/Ic_block_inactive';
 import MemoIc_comment from '../../assets/icons/Ic_comment';
 import MemoIc_share from '../../assets/icons/Ic_share';
 import Memoic_globe from '../../assets/icons/ic_globe';
+import MemoIc_senddm from '../../assets/icons/ic_send_dm';
+import SendDMAnonBlock from '../../assets/icons/images/send-dm-anon-black.svg';
+import SendDMBlack from '../../assets/icons/images/send-dm-black.svg';
+import ShareBlack from '../../assets/icons/images/share-black.svg';
+import {Context} from '../../context';
+import useDMMessage from '../../hooks/core/chat/useDMMessage';
+import useCreateChat from '../../hooks/screen/useCreateChat';
+import {colors} from '../../utils/colors';
+import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
+import dimen from '../../utils/dimen';
+import {normalizeFontSize} from '../../utils/fonts';
 import {FONTS} from '../../utils/theme';
+import BottomSheetMenu from '../BottomSheet/BottomSheetMenu';
 
 const Footer = ({
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new imports and the Footer component's props have been extended to include `onPressScore` and `isShowDM`. This seems fine, but it would be good to ensure that these new props are being passed correctly from the parent components and that they are used appropriately within this component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->